interview-questions.md

```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-Your answer:
+Your answer: A conditional statement is a set of conditions or actions you want to be met.
 
-Researched answer:
+Researched answer: A conditional statement is a command or a decision tree that make decisions to perform the different actions that we want to happen.
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is coding software. The difference between git and Github is that git is local and Github is remote. 
 
-Researched answer:
+Researched answer: Git is a free software system used by coders. The difference between git and Github is that Github is an online platform whereas git is only local to user's computer. Github is used for sharing and working on code remotely with another coder to work on the same code.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: True or false operators will return a boolean value.
 
-Researched answer:
+Researched answer: Booleans are data types. They give true or false values are not strings. They do not need quotations, you can just give the value of true or false.
```

Comment:
Exactly, booleans are data types and either true or false. There are some JS operators that will return a boolean value, can you think of any? *hint you used some in your conditional statement for # 1